rules/docker_config.bzl

```diff
@@ -268,18 +269,33 @@ def _docker_toolchain_autoconfig_impl(ctx):
         ("if [ -f /" + outputs_tar + " ]; " +
          "then tar -rf /extract.tar /" + outputs_tar + "; fi"),
     ]
-    print("\n== Docker autoconfig will run. ==\n" +
-          "To debug any errors run:\n" +
-          "> docker run -d <image_id> bash\n" +
-          "Where <image_id> is the image id printed out by the " +
-          ctx.attr.name + "_extract" + ".tar rule.\n" +
-          "Then run:\n>/" + install_sh.basename +
-          "\nfrom inside the container.")
+
+    # if mount_project was selected, lets mount it
+    docker_run_flags = [""]
```

Comment:
Can we move line 274 to line 279 to be closer to where the project source is setup, like somewhere near line 159 -164 so the logics to set up the source repo stay together?

Reply:
done